data/press.js

```diff
@@ -43,6 +43,14 @@ const press = [
     date: '11-23-2021',
     format: 'print',
   },
+  {
+    source: ['Capitol Weekly'],
+    href: 'https://capitolweekly.net/events/an-energy-forum-november-17-2021/',
+    title: 'Energy forum: The end of oil?',
+    authors: ['Julie Cart'],
+    date: '11-17-2021',
+    format: 'print',
```

Comment:
Should we tag this as a video?
```suggestion
    format: 'video',
```